site/content/extend/getting-started/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Getting Started"
+date: "2018-04-19T12:01:23-04:00"
+section: "extend"
+---
+
+# Getting Started
+
+Mattermost can be extended by building and installing plugins, by embedding Mattermost in another application, or by forking the codebase.
+
+## Extension Methods
+
+### Plugins (Beta)
+
+Plugins are a powerful way to add new functionality and UI to Mattermost - in an easy to maintain and update way. They are easy to build and give you more power than any other integration method.
```

Comment:
How about: `Plugins are easy to build, effortless to maintain, and offer powerful ways to add new functionality and UI to both the server and web/desktop apps, not available with any other integration method.`